doc_config/yard/templates/default/fulldoc/html/css/global.css

```diff
@@ -60,3 +60,7 @@ blockquote p {
   font-weight: inherit;
   line-height: inherit;
 }
+
+ul ul, ol ol, ul ol, ol ul {
+  margin-bottom: 1.25em;
```

Comment:
The margin-bottom here matches `p, blockquote` above and fixes lists within lists so they have proper margin:

<img width="279" alt="screen shot 2018-09-16 at 4 09 53 pm" src="https://user-images.githubusercontent.com/7371/45601499-0baa2600-b9cb-11e8-88ad-2a2d65b77f28.png">